### ISSUE TITLE
[A11y Image Review]: MHV Lab and Test Results

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#57857]
## Revisit in P1
`Not in P0`
[View related comment.
](https://github.com/department-of-veterans-affairs/va.gov-team/issues/57857#issuecomment-1681277042)
## a11y Reviewer(s)

- Sara Smith
- Riley
- Bobby

## Alt Text Handling for Medical Records Images
Labs and Tests in Medical Records will include medical images. Minimal information about these files are known at this time, until we receive the actual data. However, in an effort to be proactive, we are attempting to identify how we will be handling the alt text for these medical images. 

### Known Items
- DICOM Files will be used in some capacity
- Clinicians enter the data
- DICOM files include PI @laurwill @aliyahblackmore 

### Unknown Items
- File type of rendered items on the website
- Clinician data entry fields, access and limitations for mapping data
- Formats of the images such as x-rays, MRIs, sonograms, etc. 

### Resources
_"Photographers who have large collections of digital photographs use IPTC metadata to organize their collections. Typically, the photographer enters his name, contact information, captions, keywords, and copyright information to tag his images._ 

_Similarly, for radiology images, these fields are used to enter information such as clinical history, diagnosis, comments, and copyright information [Figure 9]. Once all the images are tagged, a search for specific images can be performed within an entire collection using the IPTC keywords. This is similar to searching images on the World Wide Web using Google Image search!_

[Source: NIH, National Library of Medicine](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3354356/)

## Possible Solutions

- Do we know how we should apply alt text to medical images on the site?
- Should it be as basic as what the image rendering is and the date? "Angle Left 3 Views March 10, 2022"
     - Then the user can refer to the details and results about the image?
     - Or, do we attempt to see if we can extract from the file's meta data?
     - Or, do we see if we can gain more insight into the clinician side, to see if there is a field to pull from?

### Research Questions
- How do Veterans typically use this information? 
- Do we have existing research on this, or will we need to collect use case information during testing? 

### Angela @AngelaFowler82 
- Alt text for Medical Images should be **date** and **purpose.**
- Accuracy of automatically generated image descriptions are not always dependable. 

## Image Example Attached
- Labs and Tests
- **Image**: Ankle left 3 views
- **Image description:** Mobile view of Va.gov website, displaying the angle left 3 views from March 10, 2022. The page also includes details and results.
- :warning: **How do we avoid non-descriptive alt text?** 
   - Meaning, if there are multiple images taken on the same date. It could get confusing. 
    - See Brian's original recommendation from MP in this ticket.

[Screenshot 2023-05-03 at 2.00.12 PM.png](https://images.zenhubusercontent.com/63dbe59975ee0eef93e952b2/196c39dc-d104-44ad-ae3b-550245dc87ee)

## Midpoint Review Feedback: X-Ray Images

- [Ticket #57260](https://app.zenhub.com/workspaces/collaboration-cycle-reviews-61d89e9e5f4c120011a0ad37/issues/gh/department-of-veterans-affairs/va.gov-team/57260)

- **Should:** For x-ray images, or any other test images that might show up --- If there's a way to have meaningful alt text that describes the image, that would be amazing. 

> **Otherwise, I think you might best be served with empty `alt` attributes rather than repeating the test name as the alt text for each image.** 
:point_up:
If there's not a way to provide screen reader users value with the alt text, I think it's probably better to get them to the download link sooner rather than later. But other a11y folks might disagree with that, so it's worth some conversation with the CAIA folks!

**Comment by sara-amanda** [comment]
## CAIA Update
### Moving to Backlog

- Will not appear in P0.
    - Instead of the `review all` link that opens the pictures (the list of them); they are going to have that **_link back to legacy_** for P0. 
- Moving to the backlog 
    - It will likely need to be incorporated in P1, devs are doing a lot of work to see what is possible and what this flow is going to look like. 
    - Flow won’t be available in staging for a while. 
    - Devs won’t focus on it until the next increments (probably Oct.) the earliest.

**Comment by coforma-terry** [comment]
Hey @sara-amanda - just chasing an update for this if you have one? Thanks!

**Comment by sara-amanda** [comment]
## Medical Records Rollout
### Last Known Order 10/3/2023 
- [x] Allergies `P0`
- [x] Vaccines `P0`
- [ ] Health conditions and vitals
- [x] Care summaries and notes `P0`
- [ ] Lab and test results 

**Source**: [Slack 10/3/2023 4:20 p.m. ET](https://dsva.slack.com/archives/C03Q2UQL1AS/p1696364438441009?thread_ts=1696362685.369579&cid=C03Q2UQL1AS)

### Phase 0 Launch Date
#### 12/3/2023 Medical Records P0 Launch Update
- October 2023 - Allergies - [Go No/Go Rollout Checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/allergies/rollout_checklist.md)
- December 2023 - Vaccines - [Go No/Go Rollout Checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/vaccines/rollout_checklist.md)
- December 2023 - Care Summaries/Notes

^ **Lab and Test Results** -`Not present in the launch list yet`

> The development and release of functionality will occur in five 'phases' to align with the release strategy of all MHV products. After gaining approval via the collaboration cycle, each 'domain' of Medical Records will enter into `Phase 0` one at a time. **After all domains are in `Phase 0`, they will then deploy together to `Phases 1-4`.** 
> - Vaccines 
> - `Lab and Test Results` 
>     - _Consisting of Chemistry/Hematology, Microbiology, Pathology results, Cardiology images and reports, Radiology images & Reports_
> - Care Summaries and Notes
>     - _Consisting of VA Notes, Admission & Discharge Summaries, After Visit Summaries_ 
> - Health Conditions, Vitals, and Allergies 
> - Downloadable Reports (consolidation of all domains into a single report)

Source: **[12/3/2023 Medical Records P0 Launch Update](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/README.md)**

cc: @coforma-terry

**Comment by sara-amanda** [comment]
@rileyorr this is an older MHV ticket, assigning to you, since you are working with that team now, instead of Angela. If you can take a look at this one?

**Comment by rileyorr** [comment]
Hi @alexiawunder and @BobbyBaileyRB! I inherited some older MHV-related CAIA tickets and was wondering if either of you could help me determine if this one should remain open and if it has any status updates. 

TLDR of this ticket is it appears to be related to alt text handling on medical records images and is blocked due to the team waiting on real data. 

Lmk if you have any questions or if another person would better know the status of this work. Thanks!

**Comment by alexiawunder** [comment]
Hi @rileyorr! The eng team is doing their research this sprint to see what we can do with images. At that point we will finally have enough information to be able to design the experience for MR images, and resurface the question of the alt text.

**Comment by rileyorr** [comment]
@alexiawunder awesome, thanks for that update! Looking forward to learning more about what the research uncovers

**Comment by rileyorr** [comment]
I'm going to move this back to in progress since the team started discussing this work again during last week's design and research collaboration sync.

**Comment by bryan-riley-va** [comment]
This is a ticket assigned to the CAIA team, so we shouldn't close it. These same issues will come back up with our new radiology collab cycle ticket, and Bobby is aware. (Lexi)

**Comment by rileyorr** [comment]
Bobby and I synced and are suggesting that if we have no metadata related to the images we should announce `Image x of x [body part name pulled from H1] view` (ex. Image 1 of 44 ankle left view). Waiting on confirmation from the team before moving this to validate so CAIA can track the work as completed.

**Comment by rileyorr** [comment]
The team is heading into staging review so the alt text suggestion should be implemented and this work can be moved to validate

cc @sara-amanda

**Comment by sara-amanda** [comment]
> [!NOTE]
> ## CAIA INTERNAL
> - [x] This ticket is linked to its parent intake ticket.
> - [x] This work has been completed
> - [x] Closing ticket
> - [x] Adding to tracker for CAIA Sprint 140
> - [x] [Comment re: Radiology in this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/57857#issuecomment-2417952044) for future work for @rileyorr shared for awareness - there is a CC ticket out there currently for Radiology #95081
